11ty/definitions/nl_NL/aziatisch-amerikaans.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Asian-American
+slug: asian-american
+defined: true
+speech: noun
+skip_in_table_of_content: true
+---
+overkoepelende term voor mensen van Aziatische afkomst (immigrant of door geboorte) die leven in en/of afkomstig zijn uit de Verenigde Staten.
+
+## Bemerkingen
+Omdat er veel landen en culturen onder deze term vallen, probeer zo specifiek mogelijk te zijn. Als je Vietnamees-Amerikaans of Chinees-Amerikaans bedoeld, gebruik die termen dan. Wees je er ook van bewust dat niet ieder Engels-sprekend Aziatisch persoon niet meteen Aziatisch-Amerikaans is. Ze zouden Aziatisch-Canadees kunnen zijn of een andere combinatie van nationaliteiten. In het algemeen is het verstandig om correct te zijn en het te bevestigen met de andere partij.
+
+## Note
+Kan afgekort worden door degenen die zich identificeren als Aziatisch-Amerikaans. Wees bewust van context en spel/schrijf het altijd volledig uit als je de term voor de eerste keer gebruikt. Als je niet Aziatisch-Amerikaans bent gebruik je bij voorkeur de volledige term.
```

Comment:
iirc it's proper to place a comma between two verbs such as "bent gebruik" -> "bent, gebruik"